datafusion/core/src/physical_plan/file_format/csv.rs

```diff
@@ -870,4 +948,66 @@ mod tests {
             }
         }
     }
+
+    async fn register_aggregate_csv_by_sql(ctx: &SessionContext, table_name: &str) {
+        let testdata = arrow_test_data();
+
+        ctx.sql(&format!(
+            "CREATE EXTERNAL TABLE {table_name} (
+                c1  VARCHAR NOT NULL,
+                c2  TINYINT NOT NULL,
+                c3  SMALLINT NOT NULL,
+                c4  SMALLINT NOT NULL,
+                c5  INTEGER NOT NULL,
+                c6  BIGINT NOT NULL,
+                c7  SMALLINT NOT NULL,
+                c8  INT NOT NULL,
+                c9  INT UNSIGNED NOT NULL,
+                c10 BIGINT UNSIGNED NOT NULL,
+                c11 FLOAT NOT NULL,
+                c12 DOUBLE NOT NULL,
+                c13 VARCHAR NOT NULL
+            )
+            STORED AS CSV
+            WITH HEADER ROW
+            LOCATION '{testdata}/csv/aggregate_test_100.csv'
+        "
+        ))
+        .await
+        .expect("Creating dataframe for CREATE EXTERNAL TABLE");
+    }
+
+    #[tokio::test]
+    async fn test_listing_table_insert_into() -> Result<()> {
```

Comment:
I recommend this be put into an `sqllogictest` rather than here (as it appears to be a simple explain plan test)